roles/common/tasks/prerequisites.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: Fail if external and local activemq are both in use
+  fail:
+    msg: Detected both external_activemq and activemq groups in use
+  when:
+    - groups.external_activemq is not empty
+    - groups.activemq is not empty
+  tags:
+    - molecule-notest
```

Comment:
I have the feeling this has been done just to allow using activemq host in the molecule inventory for tests, I think that's do-able and preferable using molecule prepare playbook.

Reply:
yes, it's a shameless workaround to relax that check just for molecule otherwise I can't provision activemq and use external activemq feature.

not sure it's really easy to do everything in the prepare playbook, I need to craft an inventory, startup an ec2 instance, run the playbook...?